src/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta charset="utf-8">
```

Comment:
This should fix 
![image](https://github.com/nbktechworld/full-stack-web-dev/assets/13649002/a69d0af3-df5f-4cd3-8fda-9a1c7f49a8bc)

Reply:
amazing !